### DISCUSSION TITLE
Build instruction

### DISCUSSION THREAD

**Discussion by ramgng** [polarfire-soc/polarfire-soc-documentation#296] [Linux build systems for PolarFire SoC]
I am using following repository for Icicle kit:
[https://github.com/linux4microchip/buildroot-external-microchip](https://github.com/linux4microchip/buildroot-external-microchip)

As per build instruction, I need to clone build root from below given source

`git clone https://git.busybox.net/buildroot -b 2022.02`

But seems this source is not available. 
Can I use [https://github.com/buildroot/buildroot/tree/2022.02.x](https://github.com/buildroot/buildroot/tree/2022.02.x) ?

**Comment by ConchuOD** [comment]
Hey,

You should be able to use that branch, yeah.
However, the above command does currently work!
2022.02 is a tag not a branch, but the -b argument to clone accepts both branches and tags.

Thanks,
Conor
